### ISSUE TITLE
make fighters/drones targetable only by antimissile guns of capital ships

### ISSUE THREAD

**Issue by bobafetthotmail** [endless-sky/endless-sky#5258]
Fighters and drones are underwhelming and in most cases a waste of money, this is well known by now.

My general idea to increase their survivability would be to make them targettable only by antimissile guns (and maybe light missiles) and of course by other fighters/drones.

This way they become the "rich man light ships" as that's really all they can do, and stop being obliterated 10 seconds after they left the hangar.

**Comment by Amazinite** [comment]
Duplicate of #2979/mentioned in #4034. Please use the issue template instead of deleting it and writing what you want. The feature request issue template has a "related issues" section for a reason, to force you to look up related search terms to your issue and see if it has already been mentioned. I see it's been a while since you've posted an issue, and in that time we've amassed quite the collection. :)

**Comment by bobafetthotmail** [comment]
I see this place has caught the same illness that affects StackExchange/Overflow, mark as duplicate to something that has the same keywords in the name and close.

> Duplicate of #2979

This is NOT a duplicate of #2979, he wants to have a better anti-fighter turret, while I want to limit the weapons that can target fighters at all.

>/mentioned in #4034.

it's barely mentioned in a huge wall of text of his main proposal about implementing some other mechanism.

**Comment by Amazinite** [comment]
> I saw both and neither was similar enough imho

You still didn't use the template, and had you done so you could have linked them so we at least know that you know of them.

> This is NOT a duplicate of #2979

The work required for both issues is the same; we'd need to allow anti-missile turrets to also damage ships (#4437), then have weapons only be able to target ships of a certain category (#2979). Just because the description is slightly different doesn't mean we should have a whole separate issue keeping track of what is essentially the same thing.

**Comment by bobafetthotmail** [comment]
> Just because the description is slightly different doesn't mean we should have a whole separate issue keeping track of what is essentially the same thing.

So, just because part of the work on the game engine is the same you are closing issues that ask to change the game in completely opposite ways as "duplicate"?

Am I the only one that finds this strange?

How does the workflow for implementing stuff work here, we add random stuff to game engine and then see what open issues fit that, and then add the feature for the users?
Or it is more like "we want to implement a specific plan" and then we decide what game engine changes are necessary.

tl:dr "duplicated issues" should be grouped by the proposal idea, not by the implementation details in the game engine. You can add tags for "needs X feature in game engine" if you really need to track all issues that need X feature in the engine.
